src/pages/Charcuterie.js

```diff
@@ -53,8 +56,18 @@ export default () => {
       <Card>
         <H2>Card Element</H2>
         <P>It can contain content. And even buttons!</P>
-        <Button>Primary</Button>
-        <SecondaryButton>Secondary</SecondaryButton>
+        <Button color="primary">Primary</Button>
```

Comment:
sometimes i think constants are good and sometimes I think man we shouldve just used strings. Wondering if you considered using constants for the colors? Not sure what I'd prefer tbh

Reply:
following up, i did actually use constants in Button.js for these strings, i'll pull those out as an import so we can use them here 
 `const buttonColors = ['primary', 'secondary', 'tertiary']`

Reply:
yep we're just using strings now (i checked some libraries and ya consts here just complicate things)